### ISSUE TITLE
Weird behaviour after DHCP lease problem (re-enabled nework connection)

### ISSUE THREAD

**Issue by tibbarytsur** [ava-labs/avalanchego#544]
**Describe the bug**

**To Reproduce**
Steps to reproduce the behavior.

**Expected behavior**

**Screenshots**

**Operating System**
Which OS you used to reveal the bug.

**Additional context**

By submitting this issue I agree to the [Terms and Conditions](https://files.avalabs.org/legal/bug_hunt_q2_2020_terms.pdf) of the [Developer Accelerator Program](https://avalabs.org/dap).

**Comment by tibbarytsur** [comment]
This morning arround 5 AM local time for some reason my node had a problem with DHCP lease being expired an unable to renew.
Arround 12:00 I noticed it (nr of connected nodes was 0), disabled and re-enabled my interface. Connectivity was OK again.
Nr of nodes was increasing.

Then arround 12:30 it happened again. I decided to switch to manual IP config and nr of nodes again increasing (to ~700).
During this whole time nothing in the logfiles (at INFO level) indicated any problems except a lack of version 1.0.3 warnings.
But then I noticed the nr of failed messages in Grafana was still high (280k).
Only after I had restarted avalanchego would the failed messages drop to near 0.

Then in the logs there were 7415 of these messages:
avalanchego/snow/networking/router/chain_router.go#387: QueryFailed(Nom39tjDi1Vf51VxuaZvJiDeQHwbyg6MU, 2oYMBNV4eNHyqk2fjjV5nVQLDbtmNJzq5s3qs3Lo6ftnC6FByM, 227491445) dropped due to unknown chain
I don't know if this is normal during shutdown or that these are backlogged logging messages that only got flushed when the restart was done.

**Comment by tibbarytsur** [comment]
Platform is Ubuntu 20 and avalanchego v1.0.2.

**Comment by Phistr90** [comment]
Sounds similar to my issue:
https://github.com/ava-labs/avalanchego/issues/483

Can you try to run in `log-level=debug`? For whatever reasons this is a workaround for me. I know this sounds silly.

**Comment by tibbarytsur** [comment]
I'll try to set loglevel to debug when I have more time. I also don't want to play too much with my node right now as everything seems to be working ok now and I'm staking and already at a low percentage level on vscout.io. 

If putting log level at debug is a workaround for you it seems that maybe there is an issue with flushing logs to the logfiles as I also noticed a large nr of log entries when restarting avalanchego.

**Comment by danlaine** [comment]
Closing for age. AvalancheGo has changed quite a lot since this issue was opened. Please open a new issue if you are still having this issue, or a similar one. Thank you!

**Comment by nyetwurk** [comment]
`--public-ip` should work with a hostname, not just a numeric IPv4 address.
